src/common/logging/Logging.h

```diff
@@ -48,26 +51,26 @@ bool IsDaemon(void);
 #define OSCONFIG_FILE_LOG_INFO(log, format, ...) __LOG_TO_FILE__(log, format, __INFO__, ## __VA_ARGS__)
 #define OSCONFIG_FILE_LOG_ERROR(log, format, ...) __LOG_TO_FILE__(log, format, __ERROR__, ## __VA_ARGS__)
 
-#define OsConfigLogInfo(log, FORMAT, ...) {\
-    if (NULL != GetLogFile(log)) {\
-        OSCONFIG_FILE_LOG_INFO(log, FORMAT, ##__VA_ARGS__);\
-        fflush(GetLogFile(log));\
-    }\
-    if ((false == IsDaemon()) || (false == IsFullLoggingEnabled())) {\
-        OSCONFIG_LOG_INFO(log, FORMAT, ##__VA_ARGS__);\
-    }\
-}\
-
-#define OsConfigLogError(log, FORMAT, ...) {\
-    if (NULL != GetLogFile(log)) {\
-        OSCONFIG_FILE_LOG_ERROR(log, FORMAT, ##__VA_ARGS__);\
-        fflush(GetLogFile(log));\
-    }\
-    if ((false == IsDaemon()) || (false == IsFullLoggingEnabled())) {\
-        OSCONFIG_LOG_ERROR(log, FORMAT, ##__VA_ARGS__);\
-    }\
-}\
-
+#define OsConfigLogInfo(log, FORMAT, ...) {\
+    if (NULL != GetLogFile(log)) {\
+        OSCONFIG_FILE_LOG_INFO(log, FORMAT, ##__VA_ARGS__);\
+        fflush(GetLogFile(log));\
+    }\
+    if (((false == IsDaemon()) || (false == IsFullLoggingEnabled())) && (true == IsConsoleLoggingEnabled())) {\
```

Comment:
As a follow-up, it would be great to move this logic into `IsConsoleLoggingEnabled(), such that these macros only read:

```
...
if (IsConsoleLoggingEnabled()) \
{\
       OSCONFIG_LOG_INFO(log, FORMAT, ##__VA_ARGS__);\
}\
...
```

Reply:
Agree. Will do this polishing fix in dev if that's all right with you.

Reply:
dev is fine